src/plugins/kernel_install/raspbian.sh

```diff
@@ -0,0 +1,33 @@
+# Kworkflow treats this script as a plugin for installing a new Kernel or
+# module in a target system. It is essential to highlight that this file
+# follows an API that can be seen in the "deploy.sh" file, if you make any
+# changes here, you must adjust install_modules() and install_kernel()
+# accordingly.
+#
+
+# Update boot loader API
+function update_raspbian_boot_loader()
+{
+  local name="$1"
+  local target="$2"
+  local flag="$3"
+
+  # https://www.raspberrypi.org/documentation/configuration/config-txt/boot.md
+
+  if grep --quiet "kernel" /boot/config.txt; then
+    sudo sed -ri "/kernel/s/=.+/=$name/g" /boot/config.txt
```

Comment:
In this function documentation, explain why do we need to update config.txt with `$name`